js/src/util.js

```diff
@@ -104,6 +105,24 @@ const Util = (($) => {
       }
     },
 
+    getTransitionDurationFromElement(element) {
```

Comment:
Please add a unit test for this method

Reply:
See https://github.com/twbs/bootstrap/pull/25218#pullrequestreview-87160376

Reply:
You can test when you have multi durations and when you have `ms`

Reply:
Unit tests are added: https://github.com/twbs/bootstrap/pull/25662/commits/05cef0a4ec9122ee7eb1b6e1d940809884881df2